python/coinbase-agentkit/coinbase_agentkit/wallet_providers/cdp_wallet_provider.py

```diff
@@ -37,6 +37,14 @@ class CdpWalletProviderConfig(CdpProviderConfig):
     mnemonic_phrase: str | None = Field(None, description="The mnemonic phrase of the wallet")
     wallet_data: str | None = Field(None, description="The data of the CDP Wallet as a JSON string")
 
+    class GasConfig(BaseModel):
+        """Configuration for gas multipliers."""
+
+        gas_limit_multiplier: float | None = Field(None, description="An internal multiplier on gas limit estimation")
+        fee_per_gas_multiplier: float | None = Field(None, description="An internal multiplier on fee per gas estimation")
+
```

Comment:
Let's define this config outside of the local scope of `CdpWalletProviderConfig`

We will want to reuse this object for adding gas configuration to [EthAccountWalletProvider](https://github.com/coinbase/agentkit/blob/a7f03b8aa7f04525e8eb921f3bfc07dabf471b02/python/coinbase-agentkit/coinbase_agentkit/wallet_providers/eth_account_wallet_provider.py#L30)